app/controllers/friendships_controller.rb

```diff
@@ -0,0 +1,20 @@
+class FriendshipsController < ApplicationController
+  def create
+    new_friend = User.where(email: params["New Friend's Email"])
+    if new_friend == []
+      flash[:notice] = 'Friend not in our system.'
+    else
+      already_friend = User.where(id: Friendship.where(user_id: new_friend[0].id).pluck(:friend_id)).or(User.where(id: Friendship.where(friend_id: new_friend[0].id).pluck(:user_id))).where(id: current_user.id)
```

Comment:
This is basically a duplicate of the really long line we had before, but with `.where(id: current_user.id)` tacked on the end.  We could shorten this and the other one with .....

Reply:
Awesome job, we could probably have this live in the model